src/client/components/pages/Courses/CoursesPage.tsx

```diff
@@ -227,6 +239,25 @@ const CoursesPage: FunctionComponent = (): ReactElement => {
     }, [setMeetingModalData, setModalButtonId]
   );
 
+  /**
+   * Takes the specified course and displays a modal to edit course notes
+   */
+  const openNotesModal = useCallback((course: CourseInstanceResponseDTO) => {
+    setNotesModalData({ course, visible: true });
+    setModalButtonId(`edit-course-notes-${course.id}`);
+  }, [setNotesModalData]);
+
+  /**
+   * Handles closing the notes modal and setting the focus back to the button
+   * that opened the modal.
+   */
+  const closeNotesModal = useCallback(() => {
+    setNotesModalData({ visible: false });
+    setTimeout(() => {
+      document.getElementById(modalButtonId)?.focus();
+    });
```

Comment:
So, the reason I used `document.getElementById` is that the `closeMeetingModal` and `closeInstructorModal` callbacks use the following code:

```js
if (modalButtonId && modalButtonId in refTable.current) {
  refTable.current[modalButtonId].focus();
}
```

This looks up `modalButtonId` in `refTable.current. I did try doing this before, but it didn't work because the button ID is never stored in `refTable.current` to begin with. As best I can determine, this value is initially populated by [`setButtonRef`](https://github.com/seas-computing/course-planner/blob/develop/src/client/components/pages/Courses/CoursesPage.tsx#L172-L178) which is passed to [`CourseInstanceTableBody`](https://github.com/seas-computing/course-planner/blob/develop/src/client/components/pages/Courses/CourseInstanceTableBody.tsx) and called [here](https://github.com/seas-computing/course-planner/blob/develop/src/client/components/pages/Courses/CourseInstanceTableBody.tsx#L108). Since the notes modal is not part of `CourseInstanceTableBody` this is never called.

Reply:
Looks like it's working now! Fixed in e7fe18c8172ac68ac87872a1e226dd359f00b226 and edf0bee252b1466117bb289d12803e3f84757d4d